core/build.gradle

```diff
@@ -71,6 +71,14 @@ dependencies {
     // Sentry
     implementation libs.sentry
 
+    // OpenTelemetry
+    implementation libs.opentelemetry.api
+    implementation libs.opentelemetry.instrumentation.annotations
+
+    // DataDog
+    implementation 'io.opentracing:opentracing-util:0.33.0'
+    implementation 'com.datadoghq:dd-trace-api:1.28.0'
```

Comment:
This should be referenced from `core/gradle/libs.versions.toml`

Reply:
Done in [6e44a94](https://github.com/osrd-project/osrd/pull/6583/commits/6e44a94021616158a7f6b8352f9e9ee79326d16e).